doc/man3/X509_ALGOR_dup.pod

```diff
@@ -36,9 +36,21 @@ values for the message digest B<md>.
 X509_ALGOR_cmp() compares B<a> and B<b> and returns 0 if they have identical
 encodings and non-zero otherwise.
 
+=head1 RETURN VALUES
+
+X509_ALGOR_dup(X509_ALGOR *alg) returns a valid B<X509_ALGOR> structure or NULL
```

Comment:
No where else do we specify the function parameters after the function name in this section.